telos-frontend/src/components/text/TextWidget.module.css

```diff
@@ -1,19 +1,19 @@
 .textInput {
-    resize: none;
-    width: inherit;
-    height: inherit;
-    background-color: transparent;
-    border: none;
-    outline: none;
+  resize: none;
+  width: inherit;
+  height: inherit;
+  background-color: transparent;
+  border: none;
+  outline: none;
```

Comment:
Were the spaces removed here due to the linter? Should there be indentation?

Reply:
This code already exists on upstream in this form.

Reply:
It is showing that you have removed the spaces

Reply:
Was informed that prettier made these changes. Commenting here to keep a record. Happy with the explanation :)